### ISSUE TITLE
for global hotkeys, suppress is trapping all keyboard input

### ISSUE THREAD

**Issue by jfayre** [moses-palmer/pynput#232]
Hello,
I'm on Windows 10. 
When I try to use the following code, suppress=True is causing all keys to block. I can't press Alt+tab to switch windows, or any other key combination.
This is with the latest pynput from pip.
For my application, I need to be able to use global hotkeys, but I need to suppress them.
import time
from pynput import keyboard

def on_activate_h():
    print('<ctrl>+o pressed')

def on_activate_i():
    print('<ctrl>+<alt>+i pressed')

h = keyboard.GlobalHotKeys({
        '<ctrl>+o': on_activate_h,
        '<ctrl>+<alt>+i': on_activate_i}, suppress = True)

h.start()

while True:
    time.sleep(30)
    break

Any idea what's going on here?
Thanks!

**Comment by jfayre** [comment]
I should also say that I've looked at the documentation. I see the parameter for passing win32 event info, but I have no idea how to implement this.
My application has multiple hotkeys that I need to catch globally and suppress from the rest of the system.
Any chance you could give me a code example?
Thanks!

**Comment by moses-palmer** [comment]
Thank you for your report.

Although not optional, this is by design. *pynput* does not yet provide an interface to the native hotkey interface of the platforms, bit rather an emulation.

This means that the hotkeys are, as you note, not blocked from other applications. Doing this would require further development.

**Comment by Babber** [comment]
Hi,
I have the same issue on Linux. And yes, it would be nice to have selective suppression of the hotkeys, but for the meantime, I have found a workaround with the help of AutoKey. @jfayre, on Windows, AutoHotKey could offer a solution. So I have defined the very same hotkeys with AutoKey, but with empty scripts, that is no action. AutoKey seems to block these from the rest of the system, but I guess, due to the particular implementation of PynPut, my Phyton script can still catch these key combinations. So far it seems to work perfectly. I hope, it helps you too, @jfayre.
